Readme.md

```diff
@@ -1,55 +1,63 @@
 ![Tracee Logo](docs/images/tracee.png)
 
-[![GitHub release (latest by date)](https://img.shields.io/github/v/release/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/releases)
-[![License](https://img.shields.io/github/license/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/blob/main/LICENSE)
-[![docker](https://badgen.net/docker/pulls/aquasec/tracee)](https://hub.docker.com/r/aquasec/tracee)
+<!-- links that differ between docs and readme -->
+[installation]:https://aquasecurity.github.io/tracee/latest/docs/install
+[docker-guide]:https://aquasecurity.github.io/tracee/latest/docker.md
+[kubernetes-guide]:https://aquasecurity.github.io/tracee/latest/kubernetes.md
+[prereqs]:https://aquasecurity.github.io/tracee/latest/docs/install/prerequisites.md
+Before moving on, please consider giving us a GitHub star ⭐️. Thank you!
 
-# Tracee: Runtime Security and Forensics using eBPF
+## About Tracee
 
-Tracee uses eBPF technology to tap into your system and give you access to hundreds of events that help you understand how your system behaves.
-In addition to basic observability events about system activity, Tracee adds a collection of sophisticated security events that expose more advanced behavioral patterns. 
-Tracee provides a rich filtering mechanism that allows you to eliminate noise and focus on specific workloads that matter most to you.
+Tracee is a runtime security and observability tool that helps you understand how your system and applications behave.  
+It is using [eBPF technology](https://ebpf.io/what-is-ebpf/) to tap into your system and expose that information as events that you can consume.  
+Events range from factual system activity events to sophisticated security events detect suspicious behavioral patterns.
```

Comment:
"Events range from factual system activity events to sophisticated security events **that** detect suspicious behavioral patterns."

Reply:
just adding the word will make the sentence easier to read